src/scripts/add-edit-proxy.js

```diff
@@ -1,198 +1,186 @@
-let idParam = Utils.urlParamsToJsonMap().id, oldProxySetting,
+const idParam = Utils.urlParamsToJsonMap().id, oldProxySetting,
```

Comment:
You cannot const _oldProxySetting_ because it is not set until later. Must use _let_.